gh-82987: Stop on calling frame unconditionally for inline breakpoints (#130493)
python/cpython: Misc/NEWS.d/next/Library/2025-02-24-01-49-11.gh-issue-82987.vHfQlG.rst

> :mod:`pdb` will always stop on calling frames when inline breakpoints like :func:`breakpoint` or :func:`pdb.set_trace` are used, regardless of whether the module matches ``skip`` pattern.

Reviewer: ```suggestion
:mod:`pdb` will always stop on calling frames when inline breakpoints like :func:`breakpoint` or :func:`pdb.set_trace` are used, ignoring the ``skip`` pattern (if any).
```

> :mod:`pdb` will always stop on calling frames when inline breakpoints like :func:`breakpoint` or :func:`pdb.set_trace` are used, regardless of whether the module matches ``skip`` pattern.

Reviewer: This is probably worth a what's new entry.

Reviewer: And a versionchange comment in the doc?

Author: Done
